Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -519,11 +519,12 @@ class BrowserViewController: UIViewController {
         super.viewDidAppear(animated)
         log.debug("BVC done.")
 
-        if profile.prefs.stringForKey(LatestAppVersionProfileKey) != AppInfo.appVersion && DeviceInfo.hasConnectivity() {
-            if let whatsNewURL = SupportUtils.URLForTopic("new-ios") {
-                self.openURLInNewTab(whatsNewURL)
-                profile.prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
-            }
+        if let whatsNewURL = SupportUtils.URLForTopic("new-ios"),
+           let latestMajorAppVersion = profile.prefs.stringForKey(LatestAppVersionProfileKey)?.componentsSeparatedByString(".").first
```

Comment:
You just need to include case 3, case 1 is already handled earlier in `AppDelegate` and case 2 is what you have here!